misc/loop/README.md

```diff
@@ -0,0 +1,42 @@
+# The portal loop :infinity: 
+
+## What is it ?
+
+It's a Gnoland node that aim to run with always the latest version of gno and never loose transactions history.
+
+For more information, see issue on github [gnolang/gno#1239](https://github.com/gnolang/gno/issues/1239)
+
+
+## How to use
+
+Start the loop with:
+
+``` sh
+$ docker compose up -d
+
+# or using the Makefile
+
+$ make
```

Comment:
add that the user should do `make snapshotter.switch-now` the first time to start the gnoland image

or, better, make it so that when you run `make` the first time the node is started?

Reply:
A portal loop auto start on launch now :)